public/js/clients/index.js

```diff
@@ -19,13 +19,17 @@ function debugPage(tab, actions) {
     if (isFirefox) {
       yield firefox.connectTab(tab.get("tab"));
       firefox.initPage(actions);
+      window.apiClient = firefox.getAPIClient();
     } else {
       yield chrome.connectTab(tab.get("tab"));
       chrome.initPage(actions);
+      window.apiClient = chrome.getAPIClient();
```

Comment:
yea, that seems right.

store and client seem like the right abstractions, maybe bound actions, but tough to say now